litani

```diff
@@ -236,6 +239,19 @@ def get_args():
         init_pars.add_argument(*flags, **arg)
 
 
+    dump_run_pars = subs.add_parser("dump-run")
+    dump_run_pars.set_defaults(func=lib.run_printer.dump_run)
+    for arg in [{
+            "flags": ["-r", "--retries"],
+            "metavar": "N",
+            "type": positive_int,
+            "default": 10,
+            "help":
+                "how many times to retry loading the run (Default: %(default)s)"
```

Comment:
What about:

`"how many times to retry loading the run in intervals of one second (Default: %(default)s)"`

just in case somebody like myself missed that bit from the `man` page?

Reply:
Done in 4ba87e5